templates/documents/general_upload.html

```diff
@@ -1,79 +0,0 @@
-
-<form method=post enctype="multipart/form-data" action="{$FORM_ACTION}" onsubmit="return top.restoreSession()">
```

Comment:
Same deal here. When I push code I push only the one file and all this baggage goes along with it out of my sight.